resteasy-client/src/main/java/org/jboss/resteasy/client/jaxrs/internal/ResteasyClientBuilderImpl.java

```diff
@@ -667,6 +667,25 @@ public ResteasyClientBuilder setFollowRedirects(boolean followRedirects) {
         return this;
     }
 
+    @Override
+    public ResteasyClientBuilder setFollowRedirects() {
+        ClientConfiguration config = new ClientConfiguration(getProviderFactory());
+        try {
+            if (config.hasProperty(ResteasyClientBuilder.PROPERTY_FOLLOW_REDIRECTS)) {
+                Object enabled = config.getProperty(ResteasyClientBuilder.PROPERTY_FOLLOW_REDIRECTS);
+                if (enabled instanceof Boolean) {
+                    this.followRedirects = (Boolean) enabled;
+                } else if (enabled instanceof String) {
+                    this.followRedirects = Boolean.parseBoolean((String) enabled);
+                }
+            }
+        } catch (Exception e) {
+            // catch possible exceptions (in this case we do not set follow-redirects at all)
+            LogMessages.LOGGER.warn(Messages.MESSAGES.unableToSetFollowRedirects(), e);
+        }
```

Comment:
This isn't quite right. We need to remove this whole method and check the property in the `build()` method. In the `build()` method it should just look something like:

```java
final Object localFollowRedirects = config.getProperty(PROPERTY_FOLLOW_REDIRECTS);
if (localFollowRedirects != null) {
    followRedirects = Boolean.parseBoolean(String.valueOf(localFollowRedirects));
}
```